django/core/management/templates.py

```diff
@@ -230,6 +231,17 @@ def handle(self, app_or_project, name, target=None, **options):
                     shutil.rmtree(path_to_remove)
 
         run_formatters([top_dir], **formatter_paths)
+        self.handle_target(name, target)
+
+    def handle_target(self, name, target=None):
+        if target is None:
+            target = os.getcwd()
+        else:
+            target = os.path.abspath(target)
+
+        # call needed to avoid windows encoding errors
+        sys.stdout.reconfigure(encoding="utf-8")
```

Comment:
Does this mean that other commands (such as `migrate`) crash on Unicode app/project names?

Reply:
@felixxm I should think so. I hit the same issue with #17838. 
I ran [some tests](https://github.com/django/django/actions/runs/7923111229/job/21632142889?pr=17838) and the 'System check identified no issues' output fails with Unicode characters (㐚砋)
I actually found that the `sys.stdout.reconfigure` patch [didn't work because of conflicts with `pytest`](https://github.com/pytest-dev/pytest/issues/4843)